networkit/cpp/io/test/IOGTest.cpp

```diff
@@ -826,6 +839,19 @@ TEST_F(IOGTest, testKONECTGraphReader) {
     ASSERT_EQ(G.weight(0, 1), 1.261404);
     ASSERT_EQ(G.weight(127, 48), 0.03050447);
 }
+
+TEST_F(IOGTest, testNetworkitBinaryWriteEmpty) {
```

Comment:
Since the test for GraphTool and NetworKit binary writer both do the same, I suggest to use the same naming scheme.